### ISSUE TITLE
Tests are not starting on M1 even build went fine

### ISSUE THREAD

**Issue by spamshaker** [goosewobbler/wdio-electron-service-example#24]
Not working on Macbook Pro M1
after executing ```pnpm i && pnpm run build && pnpm run test```

```
2023-01-15T16:44:47.032Z INFO webdriver: [POST] http://127.0.0.1:9519/session
2023-01-15T16:44:47.032Z INFO webdriver: DATA {
  capabilities: {
    alwaysMatch: { browserName: 'chrome', 'goog:chromeOptions': [Object] },
    firstMatch: [ {} ]
  },
  desiredCapabilities: {
    browserName: 'chrome',
    'goog:chromeOptions': {
      binary: '/wdio-electron-service-example/dist/mac/wdio-electron-service-example.app/Contents/MacOS/wdio-electron-service-example',
      args: [Array],
      windowTypes: [Array]
    }
  }
}
```

getting the error
```
2023-01-15T16:35:31.441Z ERROR webdriver: Request failed with status 500 due to unknown error: unknown error: no chrome binary at /dist/mac/wdio-electron-service-example.app/Contents/MacOS/wdio-electron-service-example
```

**Comment by spamshaker** [comment]
ok seems like it is generating build dist/mac-arm64 but tests looks into dist/mac/...
leaving the issue open as it may need some fixes anyways

**Comment by goosewobbler** [comment]
@spamshaker thanks, will take a look and update deps

**Comment by goosewobbler** [comment]
This is should be fixed in `wdio-electron-service` @ 3.5.1

https://github.com/webdriverio-community/wdio-electron-service/pull/53
